lambdas/src/main/kotlin/scorcerer/server/resources/League.kt

```diff
@@ -43,13 +46,61 @@ class League : LeagueApi() {
     }
 
     override fun getLeague(requesterUserId: String, leagueId: String): League {
```

Comment:
We might want to remove this endpoint but I've just kept it along with a league leaderboard one for now